utils/lib/DataSource.ts

```diff
@@ -23,13 +24,42 @@ export interface DataSourceMutationResponse {
   };
 }
 
+interface DataSourceContext extends QuickstartContext {}
```

Comment:
this might not be necessary. Wanted to keep the namespace consistent.

Reply:
Would it make more sense to have the QuickstartContext extend the DataSourceContext? That way the data source can define new context parameters without needed to modify the quickstart side of the things.

Reply:
Ah, that makes more sense.